README.md

```diff
@@ -1,36 +1,87 @@
+
+<p align="center"><img width="40%" src="docs/scispacy-logo.png" /></p>
+
+
 # SciSpaCy
-This repository contains custom pipes and models related to using spaCy for scientific documents. In particular, there is a custom tokenizer that adds tokenization rules on top of spaCy's rule-based tokenizer, and a custom sentence segmenter that adds sentence segmentation rules on top of spaCy's statistical sentence segmenter.
-
-# Usage
-## Using SciSpaCy as is
-To use SciSpaCy as is, follow these steps:
-1. Clone this repository
-1. From within this repository, run
-`./scripts/create_model_package.sh ./scispacy/models/combined_rule_tokenizer_and_segmenter`
-1. Run `python setup.py sdist`
-1. Run `pip install --user dist/scispacy-1.0.0.tar.gz`
-1. Run `pip install --user dist/en_scispacy_core_web_sm-1.0.0.tar.gz`
-
-Once you have completed the above steps, you can load SciSpaCy as you would any other spaCy model. For example: 
+This repository contains custom pipes and models related to using spaCy for scientific documents. In particular, there is a custom tokenizer
```

Comment:
scispaCy capitalization, also, ScispaCy or scispaCy?

Reply:
idk, let's just be frequentist about that